infrastructure/adminservices-test/altinn-monitor-test-rg/k6_tests_rg_loki_values.tftpl

```diff
@@ -94,8 +94,18 @@ ruler:
 
 # This exposes the Loki gateway so it can be written to and queried externaly
 gateway:
-  service:
-    type: LoadBalancer
+  ingress:
+    enabled: true
+    ingressClassName: nginx
+    hosts:
+      - host: k6.dis.altinn.cloud
+        paths:
+          - path: /
```

Comment:
Just curious, will this host be used for more services? if so, then the / prefix might be wrong to use for loki?

Reply:
At the moment no.
I might play with exposing Prometheus/AlertManager in the future too.
But the current use case is only for Loki so this config is fine for now.